server/src/repositories/person.repository.ts

```diff
@@ -276,13 +275,18 @@ export class PersonRepository {
         'asset_faces.imageHeight as oldHeight',
         'assets.type',
         'assets.originalPath',
-        'asset_files.path as previewPath',
         'exif.orientation as exifOrientation',
       ])
+      .select((eb) =>
+        eb
+          .selectFrom('asset_files')
+          .select('asset_files.path')
+          .whereRef('asset_files.assetId', '=', 'assets.id')
+          .where('asset_files.type', '=', AssetFileType.PREVIEW)
```

Comment:
```suggestion
          .where('asset_files.type', '=', sql.lit(AssetFileType.PREVIEW))
```

Do you have an opinion on this? I believe it includes in the query param in the generated sql, which is sometimes nice to see when it's not actually variable.

Reply:
That's a good change, I just didn't think to do it